README.md

```diff
@@ -1,57 +1,21 @@
 # Northcoders House of Games API
 
-## Background
+## Before use instructions
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Before interacting with the databases, you must first create the environment variables. Follow the steps outlined below. 
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+Run npm install 
 
-## Kanban
+Ensure that you have dotenv installed locally in the package.json file 
```

Comment:
Can just say install dotenv, don't need to specify package.json (this will always show the dependencies)